### ISSUE TITLE
Alternative Category Label

### ISSUE THREAD

**Issue by thunderbolt10** [amcharts/amcharts4#1413]
I have a column chart where the columns are categorised by a particular data field. 
I want to show a category label that takes a different data field to the field that is actually used to categorise the data.

In the pen example [https://codepen.io/thunderbolt10/pen/VJyjOd](https://codepen.io/thunderbolt10/pen/VJyjOd) data is categorised using the data field name"ident" but I want the label on the left to display the text of the "fruit" field. 
Is this possible?

**Comment by zeroin** [comment]
To change the category text you should set:

`categoryAxis.dataItems.template.text = "{fruit}";`

I agree it's not very intuitive :)

**Comment by thunderbolt10** [comment]
Thanks. Got this to work eventually. In my application the "fruit" field is actually called "category", and then when implementing your suggestion it will be:
`categoryAxis.dataItems.template.text = "{category}"`

This I found does not behave as intended but uses whichever field is the actual data category and not the field that is named category.
When I renamed my JSON data field to something other than "category" your suggestions works.
